orc8r/cloud/go/obsidian/swagger/v1/swagger.yml

```diff
@@ -7064,6 +7064,7 @@ definitions:
     required:
     - trace_id
     - trace_type
+    - timeout
```

Comment:
Why are we making this a mandatory parameter and not using default timeout in the absence of it ?

Reply:
+1

Reply:
Removed from required